admin_guide/sdn_troubleshooting.adoc

```diff
@@ -351,23 +351,15 @@ different pods.
 
 === The Interfaces on a Node
 
-These are the interfaces that the {product-title} SDN creates:
+These are the interfaces that the OpenShift SDN creates:
```

Comment:
FYI my original version had OpenShift @ahardin-rh updated them all to {product-title}.  Tagging her in case she protests this change.